pkg/jobrunaggregator/jobrunaggregatorlib/util.go

```diff
@@ -45,6 +45,11 @@ func WaitAndGetAllFinishedJobRuns(ctx context.Context,
 	variantInfo string) ([]jobrunaggregatorapi.JobRunInfo, []jobrunaggregatorapi.JobRunInfo, []string, []string, error) {
 	clock := clock.RealClock{}
 
+	// Polling involves some queries that can add up to substantial costs, there's little
+	// point as we have no jobs running in under 2 hours.
+	fmt.Println("waiting 2h for sub-jobs to complete before we start polling")
+	time.Sleep(2 * time.Hour)
+
```

Comment:
Do you want to add this here or look at changing

https://github.com/openshift/ci-tools/blob/2b99a76b22b0bc656632df0c7d8363afc22afde6/pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer.go#L73

and

https://github.com/openshift/ci-tools/blob/2b99a76b22b0bc656632df0c7d8363afc22afde6/pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go#L485